File: app/src/main/java/com/digicoffer/lauditor/Chat/Model/ClientRelationshipsDo.java
package com.digicoffer.lauditor.Chat.Model;

import org.json.JSONArray;
import org.json.JSONObject;

import java.util.ArrayList;
import java.util.List;

public class ClientRelationshipsDo {
    String adminName;
    String rel_id;
    boolean canAccept;
    String clientType;
    String client_id;
    String consent;
    String source = "relationship";
    private String lastMessage;
    private String lastMessageTime;
    int recyclerview_position;
    private int childCount;
    String created;
    JSONArray groups;
    String guid;
    String id;
    String unread_count;
    private boolean expanded;
    boolean isAccepted;
    boolean isClient;
    boolean isEditable;
    JSONArray matterList;
    String name;
    JSONArray Users;
    private long lastMessageTimestamp;
    ArrayList<ChildDO> childList = new ArrayList<>();
    private List<String> subItems;
    private String firmName;

    public long getLastMessageTimestamp() {
        return lastMessageTimestamp;
    }

    public void setLastMessageTimestamp(long lastMessageTimestamp) {
        this.lastMessageTimestamp = lastMessageTimestamp;
    }

    public String getSource() {
        return source;
    }

    public void setSource(String source) {
        this.source = source;
    }

    public String getLastMessage() {
        return lastMessage;
    }

    public void setLastMessage(String lastMessage) {
        this.lastMessage = lastMessage;
    }


    public String getLastMessageTime() {
        return lastMessageTime;
    }

    public void setLastMessageTime(String lastMessageTime) {
        this.lastMessageTime = lastMessageTime;
    }

    public String getFirmName() {
        return firmName;
    }

    public void setFirmName(String firmName) {
        this.firmName = firmName;
    }


    public String getRel_id() {
        return rel_id;
    }

    public void setRel_id(String rel_id) {
        this.rel_id = rel_id;
    }

    public ArrayList<ChildDO> getChildList() {
        return childList;
    }

    public void setChildList(ArrayList<ChildDO> childList) {
        this.childList = childList;
    }


    public String getUnread_count() {
        return unread_count;
    }

    public void setUnread_count(String unread_count) {
        this.unread_count = unread_count;
    }

    public JSONArray getUsers() {
        return Users;
    }

    public void setUsers(JSONArray users) {
        Users = users;
    }

    public int getRecyclerview_position() {
        return recyclerview_position;
    }

    public void setRecyclerview_position(int recyclerview_position) {
        this.recyclerview_position = recyclerview_position;
    }

    public List<String> getSubItems() {
        return subItems;
    }

    public int getChildCount() {
        return childCount;
    }

    public void setChildCount(int childCount) {
        this.childCount = childCount;
    }

    public void setSubItems(List<String> subItems) {
        this.subItems = subItems;
    }

    public boolean isExpanded() {
        return expanded;
    }

    public void setExpanded(boolean expanded) {
        this.expanded = expanded;
    }

    public String getAdminName() {
        return adminName;
    }

    public void setAdminName(String adminName) {
        this.adminName = adminName;
    }

    public boolean isCanAccept() {
        return canAccept;
    }

    public void setCanAccept(boolean canAccept) {
        this.canAccept = canAccept;
    }

    public String getClientType() {
        return clientType;
    }

    public void setClientType(String clientType) {
        this.clientType = clientType;
    }

    public String getClient_id() {
        return client_id;
    }

    public void setClient_id(String client_id) {
        this.client_id = client_id;
    }

    public String getConsent() {
        return consent;
    }

    public void setConsent(String consent) {
        this.consent = consent;
    }

    public String getCreated() {
        return created;
    }

    public void setCreated(String created) {
        this.created = created;
    }

    public JSONArray getGroups() {
        return groups;
    }

    public void setGroups(JSONArray groups) {
        this.groups = groups;
    }

    public String getGuid() {
        return guid;
    }

    public void setGuid(String guid) {
        this.guid = guid;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public boolean isAccepted() {
        return isAccepted;
    }

    public void setAccepted(boolean accepted) {
        isAccepted = accepted;
    }

    public boolean isClient() {
        return isClient;
    }

    public void setClient(boolean client) {
        isClient = client;
    }

    public boolean isEditable() {
        return isEditable;
    }

    public void setEditable(boolean editable) {
        isEditable = editable;
    }

    public JSONArray getMatterList() {
        return matterList;
    }

    public void setMatterList(JSONArray matterList) {
        this.matterList = matterList;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Chat/Model/ClientRelationshipsDo.kt
package com.digicoffer.lauditor.Chat.Model

import org.json.JSONArray
import java.util.ArrayList

class ClientRelationshipsDo {
    @get:JvmName("getAdminName")
    @set:JvmName("setAdminName")
    var adminName: String? = null

    @get:JvmName("getRel_id")
    @set:JvmName("setRel_id")
    var rel_id: String? = null

    @get:JvmName("isCanAccept")
    @set:JvmName("setCanAccept")
    var canAccept: Boolean = false

    @get:JvmName("getClientType")
    @set:JvmName("setClientType")
    var clientType: String? = null

    @get:JvmName("getClient_id")
    @set:JvmName("setClient_id")
    var client_id: String? = null

    @get:JvmName("getConsent")
    @set:JvmName("setConsent")
    var consent: String? = null

    @get:JvmName("getSource")
    @set:JvmName("setSource")
    var source: String? = "relationship"

    @get:JvmName("getLastMessage")
    @set:JvmName("setLastMessage")
    var lastMessage: String? = null

    @get:JvmName("getLastMessageTime")
    @set:JvmName("setLastMessageTime")
    var lastMessageTime: String? = null

    @get:JvmName("getRecyclerview_position")
    @set:JvmName("setRecyclerview_position")
    var recyclerview_position: Int = 0

    @get:JvmName("getChildCount")
    @set:JvmName("setChildCount")
    var childCount: Int = 0

    @get:JvmName("getCreated")
    @set:JvmName("setCreated")
    var created: String? = null

    @get:JvmName("getGroups")
    @set:JvmName("setGroups")
    var groups: JSONArray? = null

    @get:JvmName("getGuid")
    @set:JvmName("setGuid")
    var guid: String? = null

    @get:JvmName("getId")
    @set:JvmName("setId")
    var id: String? = null

    @get:JvmName("getUnread_count")
    @set:JvmName("setUnread_count")
    var unread_count: String? = null

    @get:JvmName("isExpanded")
    @set:JvmName("setExpanded")
    var expanded: Boolean = false

    @get:JvmName("isAccepted")
    @set:JvmName("setAccepted")
    var isAccepted: Boolean = false

    @get:JvmName("isClient")
    @set:JvmName("setClient")
    var isClient: Boolean = false

    @get:JvmName("isEditable")
    @set:JvmName("setEditable")
    var isEditable: Boolean = false

    @get:JvmName("getMatterList")
    @set:JvmName("setMatterList")
    var matterList: JSONArray? = null

    @get:JvmName("getName")
    @set:JvmName("setName")
    var name: String? = null

    @get:JvmName("getUsers")
    @set:JvmName("setUsers")
    var Users: JSONArray? = null

    @get:JvmName("getLastMessageTimestamp")
    @set:JvmName("setLastMessageTimestamp")
    var lastMessageTimestamp: Long = 0

    @get:JvmName("getChildList")
    @set:JvmName("setChildList")
    var childList = ArrayList<ChildDO>()

    @get:JvmName("getSubItems")
    @set:JvmName("setSubItems")
    var subItems: List<String>? = null

    @get:JvmName("getFirmName")
    @set:JvmName("setFirmName")
    var firmName: String? = null
}
